patchwork-vanilla-patches/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,27 @@
+{
+  "schemaVersion": 1,
+  "id": "patchwork-vanilla-patches",
+  "name": "Patchwork Vanilla Patches",
+  "version": "${version}",
+  "license": "LGPL-2.1-only",
+  "icon": "assets/patchwork-vanilla-patches/icon.png",
+  "contact": {
+    "issues": "https://github.com/PatchworkMC/patchwork-api/issues",
+    "sources": "https://github.com/PatchworkMC/patchwork-api"
+  },
+  "authors": [
+    "PatchworkMC"
+  ],
+  "depends": {
+    "fabricloader": ">=0.6.2",
+    "patchwork-fml": "*"
```

Comment:
Does it depend on patchwork-fml? It's not in the build gradle.

Reply:
it's an oversight, will remove it